site/docs/windows.md

```diff
@@ -58,7 +58,7 @@ To build C++ targets, you need:
         features are not installed by default.
 
     *   Install the [Visual C++ Build
-        Tools 2015 or later](http://landinghub.visualstudio.com/visual-cpp-build-tools).
+        Tools 2015 or later](http://go.microsoft.com/fwlink/?LinkId=691126).
```

Comment:
This link downloads a "visualcppbuildtools_full.exe" which seems pretty confusing to say the least. Maybe we should link the page where all the Visual C++ Build Tools are?

Reply:
I agree. Microsoft is notorious for orphaning download links.

Reply:
Done.